backend/apps/owasp/models/board_candidate_claim_evidence.py

```
@@ -122,12 +123,64 @@ def clean(self) -> None:
+                save_kwargs = {"format": img.format}
+                if img.format == "JPEG":
+                    save_kwargs["exif"] = b""
+
+                img.save(output, **save_kwargs)
+                output.seek(0)
+
+                from django.core.files.base import ContentFile
+
+                self.file = ContentFile(output.read(), name=self.file.name)
+            elif ext == ".pdf":
+                from pypdf import PdfReader, PdfWriter
```

Ruff (0.15.18)

[warning] `import` should be at the top-level of a file

(PLC0415)